### PR TITLE
Remove bgpdma for value > 2 because they have been deleted in nrn

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

**Comment by alkino** [comment]
@nrnhines do you think something on that?

**Comment by nrnhines** [comment]
It is good that it is removed.

File: runs.hoc [hunk 1]
```diff
@@ -51,17 +51,13 @@ if (runs == 5) {
...
-	  use2phase=0 doseries(4)
-	  use2phase=1 doseries(3)
 	use2interval=-1
-	mk_extra_cells(50) use2phase=0 doseries(4)
 	mk_extra_cells(0)
 	compress_bufsize=20 doseries(1)
 	compress_bufsize=40 doseries(1)
 
 	ncellpow = log(32*pc.nhost)/log(2)
 	mkmodel(ncellpow, 10000)
```
Comment (pramodk) [inline runs.hoc]: delete?
```suggestion
```

File: runs.hoc [hunk 2]
```diff
@@ -51,17 +51,13 @@ if (runs == 5) {
 	ncellpow = log(256*pc.nhost)/log(2)
 	mkmodel(ncellpow, 1000)
 	use2interval=0
```
Comment (pramodk) [inline runs.hoc]: delete?
```suggestion
```

**Review by pramodk — APPROVED** [review]
Apart from minor suggestions, LGTM

File: runs.hoc (hunk 2 above)
Comment (alkino) [reply runs.hoc]: of course ^^

File: runs.hoc [hunk 3]
```diff
@@ -50,18 +50,11 @@ if (runs == 5) {
...
 	ncellpow = log(256*pc.nhost)/log(2)
 	mkmodel(ncellpow, 1000)
-	use2interval=0
-	  use2phase=0 doseries(4)
-	  use2phase=1 doseries(3)
 	use2interval=-1
-	mk_extra_cells(50) use2phase=0 doseries(4)
 	mk_extra_cells(0)
 	compress_bufsize=20 doseries(1)
 	compress_bufsize=40 doseries(1)
```
Comment (alkino) [inline runs.hoc]:
```suggestion
```

**Comment by pramodk** [comment]
@nrnhines : this is good to merge? 

(By the way, we can move this repo inside neuronsimulator if that makes easy to maintain by multiple people.)

**Pull request merged**